README.md

```diff
@@ -27,22 +27,53 @@ Developed as a Concordia University Capstone project.
 * From the project root `cd /frontend/`
 * Run the command `npm install` to install dependencies
 
-#### Running the front end
+#### Running
 * To run the front-end, you have two options 
     * Run the command `npm start` for a non-optimized development build 
     * Run the command `npm run build` for a production build
     
-## Setting up code styling tools
+### Backend
 
-### Back-end
+#### Setup
+* Install Java 11 SDK
+    * To check that you are using the correct version:
+        * In your terminal, run `java -version`
+        * In IntelliJ, you can check File → Project Structure → Project SDK
+        
+* Install MySQL 8.0
+    * Download and install MySQL Community 8.0. Make sure you install at minimum the server and MySQL Workbench.
+    * Take note of your root user password. 
+        * If you were not prompted for a password during installation, go to the bin folder of your MySQL installation and run the command `mysqld --initialize --console`. This should initialize the database and print a randomly-generated one-time password to the console.
```

Comment:
maybe we should mention the mysql port we are using -> 3306, its pretty standard but just in incase

Reply:
Personal preference, I would not specify configuration port numbers in the README, especially not default ones